app/assets/javascripts/application.js

```diff
@@ -10,7 +10,9 @@
 // Read Sprockets README (https://github.com/rails/sprockets#sprockets-directives) for details
 // about supported directives.
 //
+//= require jquery3
 //= require rails-ujs
-//= require activestorage
```

Comment:
Why do you remove 'ActiveStorage'? It's a brand new feature in Rails 5.2 which allows adding files (images) without third party libraries like 'Carrierwave'.

http://weblog.rubyonrails.org/2017/11/27/Rails-5-2-Active-Storage-Redis-Cache-Store-HTTP2-Early-Hints-Credentials/